src/garage/sampler/pearl_worker.py

```diff
@@ -0,0 +1,99 @@
+"""Module for PEARLWorker."""
```

Comment:
did you forget to remove this file? I thought this was moved to pearl.py

Reply:
Yeah I will delete this.